GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -508,9 +508,11 @@ static void StraightenLanes(int startIndex, int lastStraightenIndex, int lastLoo
 
                         int straightenedCurrentLane = currentLane + 1;
                         int lookaheadLane = currentLane;
```

Comment:
```suggestion
                        int lookAheadLane = currentLane;
```

Reply:
> `- int lookaheadLane = currentLane;`
> `+ int lookAheadLane = currentLane;`

Then we would need to change the capitalization of the existing `lookaheadIndex`, too.

Reply:
Yes :)

Reply:
Renamed